cmd/main.go

```diff
@@ -32,6 +32,8 @@ var (
 	nodeID        = flag.String("nodeid", "", "node id")
 	runController = flag.Bool("controller", false, "run controller service")
 	runNode       = flag.Bool("node", false, "run node service")
+	username      = flag.String("username", "", "Windows username")
```

Comment:
Are these credentials for the samba server?

Reply:
I haven't tested this for Samba servers, but they're the credentials for an SMB share on Windows.

Reply:
wonder if this is more of a runtime configuration that may be different per volume/namespace/storage class? If so, might be better to provide through secrets instead of a flag on the binary

Reply:
I've implemented this using NodePublishSecrets but I'm not sure if I did it the right way. Feedback appreciated!